DatadogInternal/Sources/Models/SessionReplay/PrivacyLevel+SessionReplay.swift

```diff
@@ -0,0 +1,81 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
+#if os(iOS)
+import UIKit
+
+// MARK: - DatadogExtension for UIView
+
+/// Extension on `DatadogExtension` to provide access to `SessionReplayOverride` for any `UIView`.
+extension DatadogExtension where ExtendedType: UIView {
+    /// Provides access to Session Replay override settings for the view.
+    /// Usage: `myView.dd.sessionReplayOverride.textAndInputPrivacy = .maskNone`.
+    public var sessionReplayOverride: SessionReplayOverrideExtension<ExtendedType> {
+        get {
+            return SessionReplayOverrideExtension(self.type)
+        }
+        set {}
```

Comment:
/improvement

intentional? looks like it should be get only from the getter definition.

Reply:
It's intentional. The empty setter allows to access the individual properties without requiring additional logic in the setter itself, like this:
`someLabel.dd.sessionReplayOverride.textAndInputPrivacy = .maskAll`

This approach is used with other similar extensions in the codebase (e.g., [DatadogExtended](https://github.com/DataDog/dd-sdk-ios/blob/597c06ea9bfd777b62879678d1b37461a7bb2a29/DatadogInternal/Sources/Extensions/DatadogExtended.swift#L36)).